### DISCUSSION TITLE
How to create new app without EXPO?

### DISCUSSION THREAD

**Discussion by adirzoari** [infinitered/ignite#2609] [Q&A]
I want to start a new project with React Native CLI without any Expo installation. When I choose the Bare option, it still installs Expo alongside React Native CLI.

**Comment by frankcalise** [comment]
Ignite always incorporates the Expo dependencies. Bare just describes a workflow where you want to manage the native code after it is generated in the ios/android directories.

You could look at something like this repo example, which has Expo entirely stripped out (the only feature that wasn't baked in was localization yet): https://github.com/frankcalise/ignite-vanilla-template

But there is no CLI associated with that, as it's just an example
